### DISCUSSION TITLE
What is a "Scotch Bonnet"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#289] [Galaxy Trivia] [answered]
**Hint:** 🔥🌶️

**Comment by thuongtruong1009** [comment]
A Scotch Bonnet is a type of chili pepper that is commonly used in Caribbean cuisine. It is also known as the Caribbean red pepper or the Bahama Mama pepper. The Scotch Bonnet is small and round with a smooth, shiny skin and ranges in color from green to yellow to bright red when ripe. It is considered to be one of the hottest peppers in the world, with a Scoville heat rating of between 100,000 to 350,000 units, which is similar to the heat level of a habanero pepper. The Scotch Bonnet is often used in sauces, marinades, and jerk seasoning blends to add both heat and flavor to dishes.

**Comment by darko5r** [comment, marked as answer]
Scotch bonnet is a variety of chili pepper named for its supposed resemblance to a Scottish tam o' shanter bonnet. It is ubiquitous in West Africa as well as the Caribbean. Like the closely related habanero, Scotch bonnets have a heat rating of 100,000–350,000 Scoville units.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/darko5r?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
